beacon-chain/execution/service.go

```diff
@@ -787,23 +783,23 @@ func validateDepositContainers(ctrs []*ethpb.DepositContainer) bool {
 
 // Validates the current powchain data is saved and makes sure that any
 // embedded genesis state is correctly accounted for.
-func (s *Service) ensureValidPowchainData(ctx context.Context) error {
```

Comment:
Because we are returning something now, I would change the name of the function. `validPowchainData` maybe?

Reply:
good idea